### PR TITLE
enhancement: redesign nav bar

### LINKED ISSUE #1029: Redesign Nav Bar
* Use https://www.prisma.io/ as an example
* Remove Sign In Button
* Products, Use Cases, Resources, Open Source, Pricing

Screen Shot 2019-07-30 at 10 51 45 AM

### PR DESCRIPTION
**Pull request by ssspear**
Fixes #1029 

Adding you to my draft PR as I work on the nav bar redesign. Please offer feedback when you have the chance. Thanks!

### CODE REVIEW

**Comment by marbemac** [comment]
Could you toss a screenshot in here of the relevant bits when you have a moment?

**Comment by ssspear** [comment]
@marbemac Example of updated menu categories and new dropdown. Can add more screenshots if you'd like to see more. I still want to add in something that shows which menu category you're viewing when the dropdown pops up, but I'm not sure what would be best for that 

Screen Shot 2019-08-02 at 2 01 24 PM

**Comment by lottamus** [comment]
@marbemac theres a deploy preview link in the PR checks too: https://deploy-preview-1033--stoplightio.netlify.com/

enhancement__redesign_nav_bar_by_ssspear_·_Pull_Request__1033_·_stoplightio_stoplight_io

**Comment by philsturgeon** [comment]
I like that we split products and use cases, but are the open-source things not also products? Dunno if we're making it harder for folks to find the things they might have come looking for.

File: netlify/settings.yaml [hunk 1]
```diff
@@ -136,15 +156,15 @@ header:
     - href: /demo/
       isButton: true
       title: Request Demo
```
Comment (lottamus) [inline netlify/settings.yaml]: Google Optimize results suggest we update Request Demo -> Book a Demo

```suggestion
      title: Book a Demo
```

**Comment by lottamus** [comment]
Closing as I believe this has already made it on develop. Please reopen if I've missed something 🤓

**Pull request closed without merging**